README.md

```diff
@@ -1,97 +1,231 @@
 
-![lighthouse report](../assets/images/lighthouse-report.png)
+If you like this project...
 
-### [Webhint](https://webhint.io/)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-A customizable linting tool that helps you improve your site's accessibility, speed, cross-browser compatibility, and more by checking your code for best practices and common errors.
+<!-- ACKNOWLEDGEMENTS -->
 
-**NOTE:** If you are running on Windows, you need to initialize npm to create `package.json` file. 
-   ```
-   npm init -y
-   ```
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-1. Run
-   ```
-   npm install --save-dev hint@7.x
-   ```
-   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
-2. Copy [.hintrc](.hintrc) to the root directory of your project.
-3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run
-   ```
-   npx hint .
-   ```
-5. Fix validation errors.
+> Give credit to everyone who inspired your codebase.
 
-### [Stylelint](https://stylelint.io/)
+I would like to thank...
 
-A mighty, modern linter that helps you avoid errors and enforce conventions in your styles.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-1. Run
+<!-- FAQ (optional) -->
 
-   ```
-   npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-   ```
 
-   *not sure how to use npm? Read [this](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).*
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-2. Copy [.stylelintrc.json](./.stylelintrc.json) to the root directory of your project.
-3. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-4. Run `npx stylelint "**/*.{css,scss}"` on the root of your directory of your project.
-5. Fix linter errors.
-6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/cli#autofixing-errors) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+<!-- LICENSE -->
 
-## Have you already completed the Sass lessons?
+## 📝 License <a name="license"></a>
 
-If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
-a few modifications to the linter configuration files.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] This link to the MIT License in your README is broken. Kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project.

![license](https://user-images.githubusercontent.com/26597432/227262924-40c58ac3-3a14-4aa8-be78-5f246eb1ab8f.png)